pkg/ebpf/c/tracee.bpf.c

```diff
@@ -3130,16 +3098,23 @@ int BPF_KPROBE(trace_do_exit)
     return events_perf_submit(&data, DO_EXIT, code);
 }
 
-/* invoke_print_syscall_table_event submit to the buff the syscalls function handlers address from
- * the syscall table. the syscalls are strode in map which is syscalls_to_check_map and the
+/* uprobe_syscall_trigger submit to the buff the syscalls function handlers address from
+ * the syscall table. the syscalls are stored in map which is syscalls_to_check_map and the
  * syscall-table address is stored in the kernel_symbols map.
  */
-static __always_inline void invoke_print_syscall_table_event(event_data_t *data)
+SEC("uprobe/trigger_syscall_event")
```

Comment:
I left a suggestion for changes at: https://github.com/aquasecurity/tracee/pull/2049#discussion_r941866347 after this was merged.